pkg/proxy/service.go

```diff
@@ -129,6 +141,10 @@ func (sct *ServiceChangeTracker) newBaseServiceInfo(port *v1.ServicePort, servic
 	if apiservice.RequestsOnlyLocalTraffic(service) {
 		onlyNodeLocalEndpoints = true
 	}
+	onlyNodeLocalEndpointsForInternal := false
+	if apiservice.RequestsOnlyLocalTrafficForInternal(service) {
```

Comment:
honestly, these helper methods are pointless but you get credit for following the pattern :)